src/simply/simply_stage.h

```diff
@@ -66,10 +67,15 @@ typedef struct SimplyElementRect SimplyElementRect;
 struct SimplyElementRect {
   SimplyElementCommonMember;
   uint16_t radius;
+  uint16_t angle_start;
+  uint16_t angle_end;
+  uint16_t border_width;
```

Comment:
`radius` is a bit misleading on Rect since it seems like it's given a circle parameter, but it's actually the border radius.

Rather than adding the members here in Rect, you should define a new SimplyElementRadial struct that has these new fields similar to how SimplyElementText has its own struct definition. This will reduce the amount of memory Rect and Circle use.